README.md

```diff
@@ -1,3 +1,82 @@
-# Docker Image for Friendica
-
-This repository holds the official Docker Image for [Friendica](https://friendi.ca)
+# Docker Image for Friendica
+
+This repository holds the official Docker Image for [Friendica](https://friendi.ca)
+
+# What is Friendica?
+
+Friendica is a decentralised communications platform that integrates social communication. Our platform links to independent social projects and corporate services.
+
+![logo](https://cdn.rawgit.com/nupplaphil/friendica-docker/c59f235f/friendica.svg)
+
+# How to use this image
+The images are designed to be used in a micro-service environment. There are two types of the image you can choose from.

```

Comment:
Can you please stick to the general documentation rule of Friendica to only put one sentence per line. It makes finding endless sentences more easy and the diffs between textual changes get easier to read.